app/uk/gov/hmrc/agentclientrelationshipsfrontend/views/journey/ConfirmClient.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.views.html.Layout
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.views.html.components.{SubmitButton, InputRadios}
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.journey.routes
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.models.journey.AgentJourneyRequest
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.models.journey.JourneyType.AuthorisationRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits.RichRadios
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.config.Constants.ClientConfirmationFieldName
+
+@this(
+        layout: Layout,
+        govukDetails: GovukDetails,
+        formWithCSRF: FormWithCSRF,
+        inputRadio: GovukRadios,
+        submitButton: SubmitButton
+)
+
+@(form: Form[Boolean])(implicit request: AgentJourneyRequest[?], messages: Messages)
+
+@key = @{
+    "confirm-client"
+}
```

Comment:
we should also be defining a `journeyKey` for this template available from the `request.journey.journeyType.toString` value, as the content will differ for auth and deauth journeys - check the other templates that use a journeyKey to see how it's done